src/safeposix/syscalls/fs_calls.rs

```diff
@@ -1193,6 +1229,8 @@ impl Cage {
         statbuf.st_blocks = 0;
     }
 
+    // helper function to populate information of socket inode object into the
+    // statbuf
     fn _istat_helper_sock(inodeobj: &SocketInode, statbuf: &mut StatData) {
```

Comment:
document the helper function aswell

Reply:
So I have a couple of thoughts here:
1. The helper functions themselves are pretty straightforward since they are just populating fields
2. the definitions of each of those fields and the possible values they can take are pretty long and can lead to over-documentation
3. Adding the definitions of each field is probably out of the scope of the helper functions, and could maybe (but again probably not) be handled in the object definition of the inodes itself instead of here.

A better way here would be to link the man page for inodes. That should provide a better experience for folks and keep things clean IMO.

Reply:
that sounds good.